examples/sparseresunet.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
```

Comment:
We could combine the two examples and rename them as `backbones.py`. In the implementation, you could iterative over all the available backbones and output their feature shapes.